index.js

```diff
@@ -608,6 +616,10 @@ let normalizeDefinitions = (dependencies) => {
                     definition.dependencies = extend({}, parent.dependencies, definition.dependencies);
                     definition.dependencies = omitBy(definition.dependencies, value => value == null);
                 }
+
+                if (parent.static) {
+                    definition.static = extend({}, parent.static, definition.static);
```

Comment:
here is no any deps anymore. all functions like `extend`, `omit`, `find` and etc. implemented in a top of file.